northstar-tests/tests/console.rs

```diff
@@ -51,6 +51,54 @@ async fn api_version() -> Result<()> {
     Ok(())
 }
 
+/// Expect the connection to be closed if a request with a too long line is sent.
+#[runtime_test]
+async fn too_long_line() -> Result<()> {
+    let timeout = Duration::from_secs(10);
+    let io = UnixStream::connect(&northstar_tests::runtime::console_full().path()).await?;
+    let mut client = api::client::Client::new(io, None, timeout).await?;
+
+    // The default json line limit is 512K
+    let container = Container::try_from("hello-world:0.0.1").unwrap();
+    let mount = iter::repeat(container).take(100000).collect();
+
+    match client.request(model::Request::Mount(mount)).await {
+        Ok(_) => panic!("expected IO error"),
+        Err(_) => Ok(()),
+    }
+}
+
+/// Invalid install request
+#[runtime_test]
+async fn invalid_install_request() -> Result<()> {
+    let timeout = Duration::from_secs(10);
+    let io = UnixStream::connect(&northstar_tests::runtime::console_full().path()).await?;
+    let mut client = api::client::Client::new(io, None, timeout).await?;
+
+    match client
+        .request(model::Request::Install("mem".into(), 999999999))
+        .await
+    {
+        Ok(_) => panic!("expected IO error"),
+        Err(_) => Ok(()),
+    }
+}
+
+/// Stale install request that shall timeout after some seconds
+#[runtime_test]
+async fn stale_install() -> Result<()> {
+    let timeout = Duration::from_secs(10);
+    let io = UnixStream::connect(&northstar_tests::runtime::console_full().path()).await?;
+    let mut client = api::client::Client::new(io, None, timeout).await?;
+
+    let response = client
```

Comment:
Setting the stream timeout would require this to be tuneable in the runtime. We could add all those config values to the runtime configuration - otherwise there's no way for a test to set those. I'll add a ticket to think about moving constants to the config.

Reply:
#608